quint/src/cliCommands.ts

```diff
@@ -480,6 +485,20 @@ export function runSimulator(prev: TypecheckedStage):
     }
 }
 
+/**
+ * Verify a spec via Apalache.
+ *
+ * @param prev the procedure stage produced by `typecheck`
+ */
+export function verifySpec(prev: TypecheckedStage):
+  CLIProcedure<VerifierStage> {
+  const verifier = { ...prev, stage: 'verifying' as stage }
+  const _mainArg = prev.args.main
+  const _mainName = _mainArg ? _mainArg : basename(prev.args.input, '.qnt')
```

Comment:
If simulation must stay bound to this command, then we should verify that flags which only make sense with simulation are not used with `bmc`.